src/ontology/edits/oeo-physical.omn

```diff
@@ -3008,6 +3058,15 @@ Class: state_of_matter
         <http://purl.obolibrary.org/obo/BFO_0000019>
     
     
+Individual: <http://openenergy-platform.org/ontology/oeo-physical/geogenic>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "x is geogenic if it is the result of geological processes."
```

Comment:
The defs of geogenic and fossil are too close at the moment. What is the difference?

Reply:
That's true.

I'd suggest the following change of the definition for fossil:
_x is fossil if it was created **from organic material** by **geolocial** processes lasting thousands or millions of years._

This would represent what we've discussed: Everything with fossil origin is also from geogenic origin but not everything from geogenic origin is fossil.

Reply:
I implemented that def now